utest/parsing/test_statements.py

```diff
@@ -37,7 +37,8 @@
     Comment,
     EmptyLine
 )
-from robot.utils.asserts import assert_equal
+from robot.utils.asserts import assert_equal, assert_true
+from robot.utils.robottypes3 import type_name
```

Comment:
Use `from robot.utils import type_name` instead. We generally import from package level when possible and the above isn't Python 2 compatible.